### DISCUSSION TITLE
Physical model sent by API and eNotice2 Web UI

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#696] [Q&A] [answered]
We have seen that the physical models sent via API, when downloaded in eNotices2, have modified fields.

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/483acbae-9674-462b-ba41-8455273a6201)

So when trying to render them in eNotices2 they do not appear correctly.

Will the same happen in production?

What is the reason for this manipulation of the XML (physical model)?

Thanks in advance

**Comment by ferrakaop** [comment, marked as answer]
As reminded at the last worshop, eSenders submitting notices via the eNotices2 API should not attempt to use the eNotice2 user interface to interact with them. The application has not been designed to handle the same notice through two different interfaces.  If your notices was fine when you submit it via API, then it will be published fine, regardless of how the eNotices2 user interface tries to render it. The TED systems publish the valid XML as submitted via API - the only difference is the addition of the publication section (<efac:Publication>) with the OJS issue, publication date and publication number on TED.

For other questions about the eNotices2 application, please contact the TED helpdesk at ted@publications.europa.eu - discussions here should focus on the technical issues of the SDK.

**Comment by AlexAS81** [comment]
And from where can we download the physical model that we have sent through the API (when the notice is not published correctly)?
